src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -42,7 +42,8 @@
             </div>
             <div class="btn-bar toolbar">
                 <div class="pull-left-sm">
-                    <button type="submit" class="btn btn-primary save-btn" data-bind="enable: isSaveEnabled, css: { 'btn-spinner': isSaving }, visible: !inReadOnlyMode()">
+                    <button type="submit" class="btn btn-primary save-btn" data-bind="enable: isSaveEnabled, css: { 'btn-spinner': isSaving }, visible: !inReadOnlyMode()"
+                            title="Save changes. A revision will be created only if 'Settings | Document Revisions' is configured.">
```

Comment:
I'm not sure I understand this title

Reply:
Well, I added this because:

Now we have the 'Create Revision' Button, which is very visible/noticeable, in the Properties pane. 
Un-aware/new users might think this button is the only way to create a revision.
I sort of wanted this title to achieve both the following:
  1. Make them aware that a config option also exists
  2. Clarify & point out the difference between the 2 revision creation options (as the new button also has a tooltip)

Either way - this title can be always removed if it gets too annoying......

Reply:
Removed - as discussed